grafana/dashboards/k6-prometheus.json

```diff
@@ -2014,13 +2042,13 @@
     ]
   },
   "time": {
-    "from": "now-5m",
```

Comment:
@Sea-you please, do not change this option.

Reply:
fixed